Update R description (#550)
greenelab/covid19-review: content/07.pathogenesis.md

#### Reproduction Number and Dynamics of Transmission

Disease spread dynamics can be estimated using R~0~, the basic reproduction number, and R~t~, the effective reproduction number. 
Thus, accurate estimates are crucial to understanding the dynamics of infection and to predict the effects of different interventions.

> R~0~ and the timescale of infection (measured by the infectious period and the exposed period) govern population-level epidemic dynamics, with R~0~ being one of most critical epidemiological parameters [@isbn:9780691116174]. R~0~ is the average number of new (secondary) infections caused by one infected person, assuming a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x]. 

Reviewer: 2 sentences in line 51 here --> separate into 2 lines

#### Reproduction Number and Dynamics of Transmission

Accurate estimates of the reproduction number of a virus are crucial to understanding the dynamics of infection and to predict the effects of different interventions.

> The basic reproduction number and the timescale of infection (measured by the infectious period and the exposed period) govern population-level epidemic dynamics, with R~0~ being one of most critical epidemiological parameters [@isbn:9780691116174].

Reviewer: Note that you can ignore at this point for interest of time.
At this point it just makes sense to go with whatever you have time to go through, but just an optional note I'll re-explain why we had the sentence:
"The basic reproduction number and the timescale of infection (measured by the infectious period and the exposed period) describe population-level epidemic dynamics, with R0 being one of most critical epidemiological parameters [@ISBN:9780691116174]."
It's almost exactly what a very good modern infectious disease modelling textbook says at the very beginning. Every overview talks about R but I was thinking mentioning the timescale of infection would also make sense. If you just have R, all you know about the dynamics is whether the epidemic grows or doesn't (based on if R is greater than 1 or less than 1) without knowing fully how long it takes to grow if it does grow. If you have both R (unitless) & the timescale of infection then you have the dynamics with a scaled notion of time -- i.e. you can describe the dynamics with units. Maybe people just know this as soon as they think through it or read details and they talk about things related to the timescale of infection a lot in other contexts, so that's why it's not important to mention. (It depends on whether you are interested in shape of dynamics only which is major point or also with respect to time units)

Author: No I see the importance, we can put it back in :+1:

> R~0~ and the timescale of infection (measured by the infectious period and the exposed period) govern population-level epidemic dynamics, with R~0~ being one of most critical epidemiological parameters [@isbn:9780691116174]. 

Reviewer: ```suggestion
R~0~ and the timescale of infection (measured by the infectious period and the exposed period) govern population-level epidemic dynamics, with R~0~ being one of the most important epidemiological parameters [@isbn:9780691116174]. 
```

Reviewer: we need this correction for grammar & because R0 is important in multiple ways. (At the very beginning when I offered suggestions for this section I put the basic concepts in my own words without reference because they are basic to field/research I've done, but then it was decided that it would make more sense to use references even for basic things because this manuscript is multidisciplinary and to shorten discussion at the beginning I just said I'd put things very close to original text of references which is fine if it's basic math or biology that cannot be said in multiple ways exactly but not as great if there are multiple ways to say it; I tend to have compound sentences which grammatically means more than 1 clause & when it was suggested for another biology section that for stylistic choices I break up sentences into basically 1-2 clauses each a biology method ended up being cut to 1 clause instead of 2-3 clauses of original suggestion which made it look very close to the reference used (it was exact biology terms so it was okay but still I preferred a different style) and also when compound sentences get so broken up they sometimes get far from the reference they belong to -- fortunately I caught that upon rereading the manuscript (I noticed it was so biologically exact & 1 sentence that it sounded more like something you'd read in a Methods section) so I fixed it to a shorter clearer (more basic/common terms) description in my own words; after seeing that I went back to this section (and other sections) to double-check if this might have occurred early on as well and caught the 2nd clause of the above sentence as being similar to the reference without necessity for exactness.  I never experienced this before (neither in drafts or final versions) -- probably because of the multi-author-multi-edits process without going back to check references; thankfully I double(+)-check writing and code.

Reviewer: @LucyMcGowan @rdvelazquez please incorporate edit above. it's straightforward correction. thanks!

Reviewer 2: The correction was incorporated here: https://github.com/greenelab/covid19-review/pull/550/commits/d37872a5cee09c157a54e75786989c5dd066f9a5.

> Thus, accurate estimates are crucial to understanding the dynamics of infection and to predict the effects of different interventions.

Reviewer: I'm not sure what the problem is with leaving the sentence:
"The basic reproduction number and the timescale of infection (measured by the infectious period and the exposed period) govern population-level epidemic dynamics, with R0 being one of most critical epidemiological parameters [@ISBN:9780691116174]."
It's almost exactly what a very good modern infectious disease modelling textbook says at the very beginning. Everyone talks about R but just because they often miss mentioning the timescale of infection doesn't mean that we should ignore it. If you just have R, all you know about the dynamics is whether the epidemic grows or doesn't (based on if R is greater than 1 or less than 1) without knowing fully how long it takes to grow if it does grow. If you have both R (unitless) & the timescale of infection then you have the dynamics with a scaled notion of time -- i.e. you can describe the dynamics with units.
(Anyways, at this point it just makes sense with whatever we have time to go through.)

> R~0~ is one of most critical epidemiological parameters [@isbn:9780691116174]; R~0~ is the average number of new (secondary) infections caused by one infected person, assuming a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x]. 

Reviewer: typo: missed "the" so change to "...one of the most..."

> A simple mechanistic model used to describe infectious disease dynamics is a susceptible-infected-recovered (SIR) state-transition model. 

Reviewer: To make sure we are better understood, better to remove "state-transition" from this sentence. I think "state-transition" model may imply something more specific. SIR model is usually differential equations model which in the infectious disease modeling literature is called "compartmental model". "state-transition" sort of sound like Markov model terms to me. Sometimes people will use SIR as a schematic to refer to other types of models which might be markov or other forms of simulations or they may just use the term SIR to refer to the conceptual idea of categorizing people as S or I or R without specifying the type of model. But I don't think the default interpretation is "state-transition".

> A simple mechanistic model used to describe infectious disease dynamics is a susceptible-infected-recovered (SIR) state-transition model. 

Reviewer: Question/clarification for @LucyMcGowan and please excuse if this is nonsense because I'm not great at jargon & I'm not taking the time to think this through carefully, so this is just me double-checking. Would you double-check that "state-transition" is a term general or specific enough to apply here. I can guess what you mean, but I wanted to double-check that nothing is implied that's too specific or general.
-basic SIR model is usually differential equations model (deterministic, ~continuous variables corresponding to S & I & R, ~continuous time) which in the infectious disease modeling literature is called "compartmental model"
If I go through it with pencil and paper, I can derive relations based on assumptions, limits, and probability theories but I'm not gonna do that right now, so I just wanted to double-check that the term you used applies to the description I put here for basic SIR model. (your terms reminded me of Markov models but again I'd have to go through pencil-and-paper to remind myself of the forms I learned & I know there are also variations.)
-Sometimes people will use SIR as a schematic to refer to other types of models which might have mathematically different features/structures/assumptions or they may just use the term SIR to refer to the conceptual idea of categorizing people as S or I or R without specifying the type of model. But I'm wondering if this relates to how "state-transition model" can/would most likely be interpreted? (It might depend on which fields you read the most, not sure if what I'm writing is nonsense because i don't follow jargon myself but I'm just double-checking)

Author: Hmm, in my group we do refer to these as state-transition models (ie the states: S, I, R, the transitions $\beta$, $\gamma$) BUT I come from a biostatistics background, so it's possible that it is really not common elsewhere. I'll change it to compartmental model :+1:

> In this formulation individuals move through three states: susceptible, infected, and recovered; two parameters, $\gamma$ and $\beta$, are specified, the rate at which the infectious recover, and the infection transmission rate, respectively. 

Reviewer 2: This sentence read awkwardly for me. A potential rewording like below may help:

``` suggestion
In this formulation, individuals move through three states: susceptible, infected, and recovered; two parameters, $\gamma$ and $\beta$, specify the rate at which the infectious recover, and the infection transmission rate, respectively. 
```

Reviewer: I'm not sure what the problem is with leaving the sentence:
"The basic reproduction number and the timescale of infection (measured by the infectious period and the exposed period) govern population-level epidemic dynamics, with R~0~ being one of most critical epidemiological parameters [@isbn:9780691116174]."
It's almost exactly what a very good modern infectious disease modelling textbook says at the very beginning. Everyone talks about R but just because they often miss mentioning the timescale of infection doesn't mean that we should ignore it. If you just have R, all you know about the dynamics is whether the epidemic grows or doesn't (based on if R is greater than 1 or less than 1). If you have both R (unitless) & the timescale of infection then you have the dynamics with a scaled notion of time -- i.e. you can describe the dynamics with units.

Reviewer: See previous comment, the way this sentence is phrased implies different types of models (e.g. Markov) than the most commonly referred to model when using term SIR (e.g. differential equation or "compartmental". So we can generalize better, you can use this sentence instead:
"In this formulation people are categorized as either susceptible, infected..."

In this simple formulation, R~0~ is estimated as the ratio of $\beta$ and $\gamma$.[@doi:10.1126/science.abb5659; @isbn:9780691116174]. 

> A pathogen can invade a susceptible population if R~0~ > 1 [@isbn:9780691116174; @isbn:9780199209996].

Reviewer: typo. You missed the "only" -- without the "only" then you leave open the wrong possibility that it can invade a susceptible population when R0<1 as well.
Correct sentence (as before):
"A pathogen can invade a susceptible population if R~0~ > 1 [@isbn:9780691116174; @isbn:9780199209996]."

> The spread of an infectious disease at a particular time t can be quantified by R~t~, the effective reproduction number, which assumes that part of the population may already be infected or that mitigating interventions were put into place. 

Reviewer 2: Possible clarification:
``` suggestion
The spread of an infectious disease at a particular time t can be quantified by R~t~, the effective reproduction number, which assumes that part of the population has already recovered (and thus gained immunity to reinfection) or that mitigating interventions were put into place.
```

The spread of an infectious disease at a particular time t can be quantified by R~t~, the effective reproduction number, which assumes that part of the population has already recovered (and thus gained immunity to reinfection) or that mitigating interventions were put into place.
For example, if only a fraction S~t~ of the population is still susceptible, R~t~ = S~t~ x R~0~.
When R~t~ is greater than 1, an epidemic grows (i.e., the proportion of the population that is infectious increases); when R~t~ is less than 1, the proportion of the population that is infectious decreases.
R~0~ and R~t~ can be estimated directly from epidemiological data or inferred using mathematical modeling.
Modeling approaches are typically based upon a classic epidemiological model structure: the SIR model and its extensions [@https://doi.org/10.1098/rspa.1927.0118; @doi:10.1038/280361a0]. 

> In the context of SARS-CoV-2, more complex modified susceptible-exposed-infectious-recovered (SEIR) models are commonly used.[need citations]

Reviewer: ```suggestion
In the context of SARS-CoV-2, more complex modified susceptible-exposed-infectious-recovered (SEIR) models are commonly used; for example, some models incorporate  population age-structure and contact patterns among different age groups [@doi:10/ggp3xq] or spatial aspects and population movement [@doi:10.1073/pnas.2004978117], and in general possible infectious disease model complexities include various heterogeneities, different biological scales, temporal forcing, spatial structure, and stochasticity [@isbn:9780199209996].
```

Reviewer: @LucyMcGowan @alavendelm This was to give some citations & examples of possible model complexities. I used both a couple articles I read as examples since the sentence mentioned SARS-CoV-2 and a general ~textbook/reader in the field to cover the various possibilities; if you were thinking of other aspects or saw different types of models in the articles you read then you can adjust the sentence if needed. I wasn't sure exactly how to approach this myself because there are roles for models of different levels of complexities and sometimes the types of models that get used changes or increases over time. Whatever you guys thinks is good enough :-)

Reviewer 2: @LucyMcGowan this looks like the only suggestion that hasn't been incorporated yet. I saw your 👍 emoji so figured you wanted to commit it but wanted to make sure.

Estimates of R~0~ for COVID-19 lie in the range R~0~=1.4-6.5 [@doi:10/ggjvr7; @doi:10.1093/jtm/taaa021; @doi:10.1126/science.abb3221].
Variation in R~0~ is expected between different populations and the estimated values of R~0~ discussed below are for specific populations in specific environments. 
The different estimates of R~0~ should not necessarily be interpreted as a range of estimates of the same underlying parameter.

> <!--I am not quite sure about this next paragraph. Most sophisticated models now incorporate "data" in their SEIR formulation, so I'm not sure it's accurate to draw the distinction.-->

Reviewer 2: I also wasn't sure about that distinction. @yemarshall or @alavendelm may be able to explain their thinking for this.

> Data-derived estimates (i.e. those that do not incorporate SIR-type models into their analysis) typically predict lower values of R~0~.

Reviewer 3: I agree wording is awkward since SEIR models use data. Suggestion below. also fine to just remove the sentence.
```suggestion
Estimates of R~0~ that relied only on data and did not use a SEIR model in their analyses typically predicted lower values.
```

> For data-derived estimates, in one study of international cases, the predicted value is R~0~=1.7 [@doi:10.1101/2020.03.21.20040329], in China (both Hubei province and nationwide), the value is predicted to lie in the range R~0~=2.0-3.6 [@doi:10.2139/ssrn.3524675; @doi:10.1101/2020.01.25.919787; @doi:10/ggjvr7], and on a cruise ship where an outbreak occurred, predicted R~0~=2.28 [@doi:10.1016/j.ijid.2020.02.033].

Reviewer 3: ```suggestion
In one study of international cases, the predicted value is R~0~=1.7 [@doi:10.1101/2020.03.21.20040329], in China (both Hubei province and nationwide), the value is predicted to lie in the range R~0~=2.0-3.6 [@doi:10.2139/ssrn.3524675; @doi:10.1101/2020.01.25.919787; @doi:10/ggjvr7], and on a cruise ship where an outbreak occurred, predicted R~0~=2.28 [@doi:10.1016/j.ijid.2020.02.033].
```

SEIR model-derived estimates of R~0~ range from 2.0 - 6.5 in China [@doi:10.3390/jcm9020462; @doi:10.1101/2020.01.27.20018952; @doi:10.1101/2020.01.23.916726; @doi:10.1101/2020.01.23.20018549] to R~0~=4.8 in France [@doi:10.1101/2020.03.22.20040915].

> Using the same model as for the French population, a study estimated R~0~=2.6 in South Korea [@doi:10.1101/2020.03.22.20040915], which is consistent with other studies [@tag:Park2020_distancing].

Reviewer 2: 👍

In Europe, analysis of 11 countries inferred the dynamics of R~t~ over a time range from the beginning of the outbreak until March 28, 2020, by which point most countries had implemented major interventions (such as school closures, public gathering bans, and stay-at-home orders) [@doi:10.25561/77731].
Across all countries, the mean R~t~ before interventions began was estimated as 3.87; R~t~ varied considerably, from below 3 in Norway to above 4.5 in Spain.
After interventions, R~t~ decreased by an average of 64% across all countries, with mean R~t~=1.43.

> The lowest predicted value was 0.97 for Norway and the highest was 2.64 for Sweden, this may be in part because Sweden did not implement social distancing measures on the same scale as other countries.

Reviewer 2: 👍

Reviewer: tiny grammar typo so not a big deal if you don't get to it:
if you want to replace parentheses with comma you need to put a "--" or a "which" so that it's not 2 independent clauses.
"The lowest predicted value was 0.97 for Norway and... 2.64... for Sweden, which may be in part because..."
or
"The lowest predicted value was 0.97 for Norway and... 2.64... for Sweden -- this may be in part because..."
